packages/babel-helper-compilation-targets/test/.browserslistrc

```diff
@@ -0,0 +1,2 @@
+[custom]
```

Comment:
nit: Let's move it to a separate folder (i.e. `test/custom-browserslists-env/.browserslistrc`) so that it will not potentially interfere with other tests.

Reply:
As I mentioned in the previous PR, I definitely think we should do this